### PR TITLE
Add Zulip and Stack Overflow contact badges

### PR DESCRIPTION
**Pull request by yurloc**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+"Join Zulip Chat", link="https://kie.zulipchat.com/#narrow/stream/232679-optaplanner"]
+
+{sonarBadge}&metric=alert_status["Quality Gate Status", {sonarLink}]
+{sonarBadge}&metric=reliability_rating["Reliability Rating", {sonarLink}]
+{sonarBadge}&metric=security_rating["Security Rating", {sonarLink}]
+{sonarBadge}&metric=sqale_rating["Maintainability Rating", {sonarLink}]
+{sonarBadge}&metric=ncloc["Lines of Code", {sonarLink}]
+{sonarBadge}&metric=coverage["Coverage", {sonarLink}]
+
 https://www.optaplanner.org/[www.optaplanner.org]
```
Comment (ge0ffrey) [inline README.adoc:19]: @yurloc Already merged, but can you move the optaplanner.org link before the badges?

Comment (ge0ffrey) [reply README.adoc:19]: Let's do the "Ask a question" and zulip link below the sonar badges? Quality comes before get help in a "getting started" journey.

Comment (ge0ffrey) [reply README.adoc:19]: Let me try editing that directly

Comment (yurloc) [reply README.adoc:19]: Already moved optaplanner.org link. Please rebase.

Yes, please fix directly as you like.

**Pull request merged**